__tests__/params.spec.ts

```diff
@@ -2,27 +2,79 @@ import { mount } from '@vue/test-utils'
 import { watch } from 'vue'
 import { getRouter } from '../src'
 
-describe('setParams', () => {
-  it('sets current route params', async () => {
-    const router = getRouter()
-    router.setParams({ userId: 12 })
-    const wrapper = mount({
-      template: `<p>{{ $route.params.userId }}</p>`,
+describe('Params utils', () => {
+  describe('`setParams`', () => {
```

Comment:
Can you remove the backticks here and on the other places, please?

```suggestion
  describe('setParams', () => {
```